library/core/src/ops/deref.rs

```diff
@@ -32,6 +62,11 @@
 /// [ref-deref-op]: ../../reference/expressions/operator-expr.html#the-dereference-operator
 /// [method resolution]: ../../reference/expressions/method-call-expr.html
 /// [type coercions]: ../../reference/type-coercions.html
+/// [box]: ../../std/boxed/struct.Box.html
+/// [refcell]: ../../std/cell/struct.RefCell.html
+/// [rc]: ../../std/rc/struct.Rc.html
```

Comment:
These links should all use intra doc links

Reply:
This is in `core` so that's not possible for Box and Rc, which are defined in `alloc`.

Reply:
then it should point to alloc and not std at least. And the ones defined in core like RefCell should point there